src/Maui/Features/Creatures/Import/Create.razor

```diff
@@ -0,0 +1,85 @@
+﻿@page "/create"
+@using TabletopPowerTools.Maui.Features.Creatures.Shared.CreatureDetail;
+@using TabletopPowerTools.Core.Models;
+
+<MudForm Model="@creature">
```

Comment:
This should probably be a view model that exists separate from a creature. Binding directly to a creature is going to cause problems if you’re wishing to have different game systems.

Reply:
I think this helps with all 3 comments, tbh. Ill start here